domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/FastTravelBetweenStopsNotice.java

```diff
@@ -25,12 +25,13 @@
 public class FastTravelBetweenStopsNotice extends ErrorNotice {
 
     public FastTravelBetweenStopsNotice(final String tripId,
-                                        final float speedkmh,
+                                        final float speedKmh,
                                         final List<Integer> stopSequenceList) {
         super("stop_times.txt", E_046,
                 "Fast travel between stops",
-                "Fast travel detected in trip:`" + tripId + "` over stop sequence:"
-                        + stopSequenceList + ". Calculated speed: " + speedkmh + "kmh",
+                String.format("Fast travel detected in trip: `%s` over stop sequence: `%s`." +
+                                " Calculated speed: `%s` kmh.",
+                        tripId, stopSequenceList, speedKmh),
```

Comment:
Nothing really, but in any other file, you put one variable per line (for clarity I assume). Maybe just do the same here to maintain the same scheme.